metadata-models/src/main/pegasus/com/linkedin/metadata/key/SchemaKey.pdl

```diff
@@ -0,0 +1,18 @@
+namespace com.linkedin.metadata.key
+
+/**
+ * Key for a Schema
+ */
+@Aspect = {
+  "name": "schemaKey"
+}
+record SchemaKey {
+  /**
+  * A unique id for the Schema.
+  * There are no rules for what constitutes a valid id for a Schema. 
+  * Valid options are:
+  * - guid("schema-registry", subject-name, version)
```

Comment:
In our organization we have plenty of JSON schemas. A best practice in JSON schemas is to include a [unique identifier](https://json-schema.org/understanding-json-schema/basics.html#declaring-a-unique-identifier) which is usually a unique URL in the organization's domain. 
So `guid(<domain>, <subject-name>, version)` would fit perfectly too.